tests/models/deformable_detr/test_modeling_deformable_detr.py

```diff
@@ -591,6 +591,18 @@ def create_and_check_model_fp16_forward(self):
         output = model(**inputs)["last_hidden_state"]
         self.parent.assertFalse(torch.isnan(output).any().item())
 
+    def create_and_check_model_bf16_forward(self):
+        model_class = DeformableDetrForObjectDetection
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        model = model_class(config)
+        model.to(torch_device)
+        model.bfloat16()
+        model.eval()
```

Comment:
The right way to load a model in as a specific dtype is using the `torch_dtype` kwarg

```suggestion
        model = model_class(config, torch_dtype=torch.bfloat16)
        model.to(torch_device)
        model.eval()
```